sound/soc/sof/Kconfig

```diff
@@ -235,5 +235,6 @@ source "sound/soc/sof/amd/Kconfig"
 source "sound/soc/sof/imx/Kconfig"
 source "sound/soc/sof/intel/Kconfig"
 source "sound/soc/sof/xtensa/Kconfig"
+source "sound/soc/sof/mediatek/Kconfig"
```

Comment:
typo in commit message: Cortex A78 cores paired with four Cortex A55 cores.

Reply:
ok. I will fix it. thanks